.storybook/themes/news.scss

```diff
@@ -1 +1,188 @@
-@use "@wpmedia/arc-themes-components/scss";
+@use "@wpmedia/arc-themes-components/scss" with (
+	$breakpoints: (
+		default: "min-width: 0",
+		desktop: "min-width: 48rem",
+	),
+	$global: (
+		"spacing-1": 0.25rem,
+		// 4px
+		"spacing-2": 0.5rem,
+		// 8px
+		"spacing-3": 0.75rem,
+		// 12px
+		"spacing-4": 1rem,
+		// 16px
+		"spacing-5": 1.5rem,
+		// 24px
+		"spacing-6": 2rem,
+		// 32px
+		"spacing-7": 2.5rem,
+		// 40px
+		"spacing-8": 3rem,
+		// 48px
+		"spacing-9": 3.5rem,
+		// 56px
+		"spacing-10": 4rem,
+		// 64px
+		"spacing-11": 4.5rem,
+		// 72px
+		"spacing-12": 5rem,
+		// 80px
+		"spacing-13": 5.5rem,
+		// 88px
+		"spacing-14": 6rem,
+		// 96px
+		"spacing-15": 6.5rem,
+		// 104px
+		"spacing-16": 7rem,
+		// 112px
+		"spacing-17": 7.5rem,
+	),
+	$alias: (
+		// Layout
+		"layout-max-width": 1600,
+		"content-max-width": 1440,
+		"content-scale-width": calc(var(--content-max-width) / var(--layout-max-width) * 100%),
+	),
+	$tokens: (
+		default: (
+			blocks: (
+				right-rail: (
+					gap: var(--global-spacing-6),
+				),
+				right-rail-navigation: (
+					position: sticky,
+					top: 0,
+					width: 100%,
+					z-index: 9,
+					margin: 0 0 var(--global-spacing-5) 0,
+				),
+				right-rail-full-width-1: (
+					margin: 0 0 var(--global-spacing-5) 0,
+				),
+				right-rail-main: (
+					gap: var(--global-spacing-6),
+					max-width: calc(var(--content-max-width) * 1px),
+					width: var(--content-scale-width),
+					margin: auto,
+				),
+				right-rail-full-width-2: (
+					margin: var(--global-spacing-6) 0 0 0,
+				),
+				right-rail-rail-container: (
+					grid-template-columns: initial,
```

Comment:
Are we still needing to use `initial` for prop values on the default viewport?
The question was asked recently and I want to make sure I have the latest info.
Thanks!

Reply:
yep this is needed for blocks. I put a note in the pr as well: 

> I included some initial in the values as the breakpoints were not allowing overriding the values. I seem to remember this being fixed. I updated the arc themes components package. I was not seeing the breakpoints updating as expected. Is there an issue with my setup? -> this is only fixed for components, not blocks

Reply:
Cool! Sorry I missed the note!